src/tests/system/tests/test_netgroups.py

```diff
@@ -108,3 +111,130 @@ def test_netgroups__add_remove_netgroup_member(client: Client, provider: Generic
     assert len(result.members) == 1
     assert "(-, user-1)" not in result.members
     assert "(-, user-2)" in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.topology(KnownTopology.Samba)
+def test_netgroup__user_attribute_membernisnetgroup_uses_group_dn(client: Client, provider: GenericProvider):
+    """
+    :title: User's 'memberNisNetgroup' attribute values are the DN of the group
+    :setup:
+        1. Create users and groups
+        2. Create a new netgroup "group" and add a member (ng1)
+        3. Create another netgroup "nested_group" and add a member (ng2)
+        4. Modify the "nested_group" to replace its members with the members of group
+        5. Start SSSD
+    :steps:
+        1. Retrieve all members of the "nested_group"
+        2. Confirm that the member directly added to "nested_group" is present
+        3. Confirm that the member from "group" is now part of "nested_group"
+    :expectedresults:
+        1. All members should be retrieved
+        2. Members directly added to "nested_group" is present
+        3. Members from group is now part of "nested_group"
+    :customerscenario: False
+    """
+    if isinstance(provider, (LDAP, Samba, AD)):
```

Comment:
It would be better to reverse the condition and fail, otherwise the test will blindly pass if IPA is added accidently.

ie

```
if not isinstance(provider, (LDAP, Samba, AD)):
    raise ValueError(...)
```

Same on other places.